src/auth/auth.service.ts

```diff
@@ -294,11 +294,29 @@ export class AuthService {
         userDto.token,
         clientIp,
       );
+      //check if the user has any unsigned cert statements
+      if (facilities.missingCertificationStatements) {
+        this.logger.error('Login Error: User has unsigned certificate statements');
+        //if the user has unsigned cert statements, we need to fail the login
+
+        if (!this.bypassService.bypassEnabled()) {
+          //terminate OIDC session
+          const apiToken = await this.tokenService.getCdxApiToken();
+          await this.oidcHelperService.terminateOidcSession(session.oidcPolicy, apiToken);
+        }
```

Comment:
In addition to terminating the CDX session, we need to also remove user infromation from the user_session table.  See `UserSessionService.removeUserSessionByUserId` method.